src/Models/Domain/CloudEdgeStatusResponse.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Middleware.Models.Enums;
+
+namespace Middleware.Models.Domain;
+//internal class CloudEdgeStatus
+//{
+//}
+
+public class CloudEdgeStatusResponse
```

Comment:
@appmdev can you move these two classes to the `CentralApi.Contracts` project? They are used only for the communication with CentralApi, so their place should be there :)